src/main/groovy/org/fundacionjala/gradle/plugins/enforce/tasks/salesforce/deployment/Deployment.groovy

```diff
@@ -35,6 +35,7 @@ abstract class Deployment extends SalesforceTask {
     public String taskFolderName
     public String taskPackagePath
     public String taskDestructivePath
+    private final String FILE_CUSTOM_COMPONENT_TRACKER_NAME = ".customComponentTracker.data"
```

Comment:
It is better to have a list to setup the default excluded files. E.G. private List<String> defaultExcludedFiles = Arrays.asList(".customComponentTracker.data")

Later it would be easy to add more files to this list.